.github/workflows/build-and-test.yml

```
@@ -25,3 +25,9 @@ jobs:
       - name: build_test
         shell: bash
         run: .github/workflows/build_and_test.sh
+      - name: fuzz_test
+        shell: bash
+        if: matrix.os[0] == 'self-hosted' && matrix.os[3] == 'jammy' # Only run on self-hosted jammy
```

Is there a reason why we run the fuzz testing on `arm` instances?
```suggestion
        if: matrix.os == 'ubuntu-22.04'
```
Guess they might be faster

---

yeah I just picked the fastest one